src/streamlink/stream/hls_playlist.py

```diff
@@ -94,7 +95,7 @@ def create_stream_info(self, streaminf, cls=None):
 
         bandwidth = streaminf.get("BANDWIDTH")
         if bandwidth:
-            bandwidth = float(bandwidth)
+            bandwidth = round(float(bandwidth), 1 - int(math.log10(float(bandwidth))))
```

Comment:
Why is bandwidth parsed as a float?

https://tools.ietf.org/html/rfc8216#section-4.3.4.2

> The following attributes are defined:
>
>      BANDWIDTH
>
>      The value is a decimal-integer of bits per second.  It represents
>      the peak segment bit rate of the Variant Stream.

Reply:
seems like it was a fix for the twitch plugin

https://github.com/streamlink/streamlink/commit/4305c8c7ca90fb328daad537bd6f4bb5dd4e20b5

https://github.com/chrippa/livestreamer/pull/1397